### DISCUSSION TITLE
为什么没有诸神匠心！！！

### DISCUSSION THREAD

**Discussion by CYSLK** [Meloong-Git/PCL#2025] [提问] [answered]
<img width="281" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/136172729/8378a789-6efe-449d-ab58-73d038a9ae36">
但是……
<img width="446" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/136172729/d60d6656-a066-428c-9b57-e9b9403fc4f4">
翻了好几页都没有……

**Comment by Chiloven945** [comment, marked as answer]
![image](https://github.com/Hex-Dragon/PCL2/assets/116699482/a6772d64-31ef-4f66-abc1-067bcc2677c6)
这个整合包并没有上传curseforge或者modrinth，所以说Pcl无法找到
请使用其他方式下载

**Comment by CYSLK** [comment]
谢谢

在 2023-06-11 14:49:50，ClovenBugle ***@***.***> 写道：



这个整合包并没有上传curseforge或者modrinth，所以说Pcl无法找到
